apps/test/netsim/NetSimRouterNode.js

```diff
@@ -234,6 +234,110 @@ describe("NetSimRouterNode", function () {
     });
   });
 
+  describe("address assignment rules", function () {
+    var router, wire1, wire2, wire3;
+
+    beforeEach(function () {
+      NetSimRouterNode.create(testShard, function (e, r) {
+        router = r;
+      });
+
+      NetSimWire.create(testShard, router.entityID + 1, router.entityID, function (e, w) {
+        wire1 = w;
+      });
+
+      NetSimWire.create(testShard, router.entityID + 2, router.entityID, function (e, w) {
+        wire2 = w;
+      });
+
+      NetSimWire.create(testShard, router.entityID + 3, router.entityID, function (e, w) {
+        wire3 = w;
+      });
```

Comment:
We shouldn't; `testShard` holds all the persistent state here and it's a local fake that gets totally recreated before every test.